dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -277,6 +277,24 @@
 			status = "okay";
 		};
 
+		dmic0: dai-dmic@10000 {
+			compatible = "intel,dai-dmic";
+			reg = <0x10000 0x8000>;
+			fifo = <0x0008 0x100>;
```

Comment:
This seems reasonable to me (again: with "me" **not** being the best reference for DTS design patterns).  But AFAICT you're only ever using the first entry of this array in the driver code.  What's does the second value mean?  Also, is there a way to construct this from a simpler "index" instead of a raw register offset?  Maybe the hardware has microphone register blocks in an array with a stride of 256 bytes?

Reply:
Yep, what does the second value means ?

Reply:
changed now to int value